api/agent/config.go

```diff
@@ -143,6 +147,9 @@ func NewConfig() (*Config, error) {
 	if _, ok := os.LookupEnv(EnvDisableDebugUserLogs); ok {
 		cfg.DisableDebugUserLogs = true
 	}
+	if _, ok := os.LookupEnv(EnvEnableDockerSyslog); ok {
+		cfg.EnableDockerSyslog = true
```

Comment:
This will enable syslog whenever the environment variable is present, regardless of its value.  Should this code check that the value in the env variable evaluates to something truthy?